File: codes/cpp/chap01/answer_A02.cpp
#include <iostream>
using namespace std;

int N, X, A[109];
bool Answer = false;

int main() {
	// 入力
	cin >> N >> X;
	for (int i = 1; i <= N; i++) cin >> A[i];

	// 全探索（変数 Answer は「既に x が見つかったかどうか」を表す）
	for (int i = 1; i <= N; i++) {
		if (A[i] == X) Answer = true;
	}

	// 出力
	if (Answer == true) cout << "Yes" << endl;
	else cout << "No" << endl;
	return 0;
}

File: codes/python/chap01/answer_A02.py
# 入力
N, X = map(int, input().split())
A = list(map(int, input().split()))
Answer = False

# 全探索（変数 Answer は「既に x が見つかったかどうか」を表す）
for i in range(N):
	if A[i] == X:
		Answer = True

# 出力
if Answer == True:
	print("Yes")
else:
	print("No")
